Docs/source/Software/Tasks.rst

```diff
@@ -2,6 +2,37 @@
 Real Time Operating System Tasks
 ********************************
 
+Initialization Task: Alex Koo
+===============================
+
+Purpose
+    Create tasks by calling "OSTaskCreate()" and provide argument specifying to RTOS how the task will be managed.
+    Initialization Task also creates the semaphores which is a programming construct that helps us achieve concurrency 
```

Comment:
If this is a definition in the glossary, I would reference it using :term: `word<term>`. If it isn't I would add it to the glossary.